src/migratus/database.clj

```diff
@@ -154,19 +164,17 @@
   driver does *not* tell you whether the table is on the current schema search
   path.)"
   [db table-name]
-  (sql/with-db-transaction
-    [t-con db]
-    (try
-      (sql/db-set-rollback-only! t-con)
-      (sql/query t-con [(str "SELECT 1 FROM " table-name)])
-      true
-      (catch SQLException _
-        false))))
+  (try
+    ;; TODO: @ieugen: do we need :rollback-only here ?
+    (let [db (connection-or-spec db)]
+      (jdbc/execute! db [(str "SELECT 1 FROM " table-name)]))
```

Comment:
I think this should be parametarized. should be a `jdbc/query` instead with `table-name` as the parameter.

Reply:
done.